workloads/network-perf/run_hostnetwork_network_test_fromgit.sh

```diff
@@ -7,7 +7,9 @@ export pairs=1
 deploy_workload
 wait_for_benchmark
 assign_uuid
-run_benchmark_comparison
 delete_benchmark
+if [[ ${COMPARE} == "true" ]]; then
```

Comment:
we need to generate csv even when COMPARE=false
it will just create current run's results csv

Reply:
Makes sense, will implement the change.